### PR TITLE
Added Range Questions and Responses

### PR DESCRIPTION
**Pull request by mytoalkoreath**
Added the Number Range Question and Responses classes to the model

### CODE REVIEW

File: src/main/java/quickndirty/minisurveymonkey/RangeQuestion.java [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+		this.max = max;
+		
+		
+	}
+	
+	public boolean addResponse(RangeResponse r) {
+		if(r.getAnswer() < min || r.getAnswer() > max) {
+			return false;
+		}else {
+			Response q = (Response)r;
```
Comment (austinjturner) [inline src/main/java/quickndirty/minisurveymonkey/RangeQuestion.java]: Is this line needed?

**Comment by austinjturner** [comment]
@mytoalkoreath, could you have a look at the `Question` and `Response` superclasses. I think this branch would be the correct place to also add the new `@JsonSubTypes.Type` annotations we'll need to map these to JSON. It should be a quick addition.

File: src/main/java/quickndirty/minisurveymonkey/RangeQuestion.java (hunk 1 above)
Comment (mytoalkoreath) [reply src/main/java/quickndirty/minisurveymonkey/RangeQuestion.java]: That was actually a typo, I'll fix it, thanks.

**Comment by mytoalkoreath** [comment]
I'll look into the annotations

**Pull request merged**